optimum/intel/generation/modeling.py

```diff
@@ -204,16 +302,13 @@ def _from_transformers(
         save_dir = TemporaryDirectory()
         save_dir_path = Path(save_dir.name)
         torch.jit.save(traced_model, save_dir_path / WEIGHTS_NAME)
+        config.torchscript = True
 
-        return cls._from_pretrained(

```

Comment:
I think we should keep `cls._from_pretrained` in order to have all the model transformation at one place (like  the 2 iterations to convert model to int8 model for IPEX)